src/test/java/seedu/address/testutil/StudentBuilder.java

```diff
@@ -145,9 +150,13 @@ public StudentBuilder withListOfSessions(List<Session> sessionList) {
         return this;
     }
 
-
+    /**
+     * Builds the student object.
+     * @return
+     */
     public Student build() {
-        return new Student(name, phone, email, address, studyLevel, guardianPhone, relationship);
+        Student s = new Student(name, phone, email, address, studyLevel, guardianPhone, relationship, sessions);
```

Comment:
Hmmm using student instead of s, might be better in this case

Reply:
Amended in next commit!